src/main/java/codacy/api/request/Endpoints.java

```diff
@@ -4,7 +4,9 @@
 
     public final static String API = "/2.0";
 
-    public final static String API_TOKEN = "api_token";
+    public final static String API_TOKEN = "api-token";
+    // This is deprecated and is kept for backward compatibility. It will be removed in the context of CY-1272
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 110).](https://app.codacy.com/gh/codacy/codacy-api-java/pullRequest?prid=5278869)